heartbeat/postfix

```diff
@@ -290,7 +297,7 @@ postfix_validate_all()
         if ocf_is_true $status_support; then
             user=`postconf $OPTION_CONFIG_DIR -h mail_owner 2>/dev/null`
             for dir in $data_dir; do
-                if ! su -s /bin/sh - $user -c "test -w $dir"; then
+                if ! $SU -s /bin/sh -l $user -c "test -w $dir"; then
```

Comment:
It should still use `-` and not `-l` here. No need to change parts that already work.

Reply:
Minor change and not really necessary you’re right. My logic was to make it clearer to understand at a glance. The man page recommends using the full —login rather than the shortcut - so I went for a middle ground

Reply:
Yeah. I would rather have it the way it has been to avoid any issues with backporting or similar (e.g. *BSD su).